### ISSUE TITLE
--inventory option is ignored

### ISSUE THREAD

**Issue by benthomasson** [ansible/ansible-runner#128]
It seems that the inventory option is not used.  After looking into the unit tests it seems they were using the default inventory with localhost and not testing the inventory option.

Here are the failing tests:

https://travis-ci.org/benthomasson/ansible-runner-1/builds/418494338

**Comment by benthomasson** [comment]
A good way to reproduce this is to turn off sshd on your machine and then run the unit tests.   They will fail.   I tried fixing this with ansible_connection=local, but it was ignored.    In ansible_runner/__main__.py I do not see where the inventory option is passed to run_options.

**Comment by benthomasson** [comment]
The inventory option is completely ignored.   The code in `ansible_runner.util.dump_artitfacts` always overwrites `private_directory/inventory/hosts` . 


This help documentation is incorrect:

```
parser.add_argument("--inventory",
  192                         help="Override the default inventory location in private_data_dir")
  193 
```

**Comment by benthomasson** [comment]
@matburt we should discuss how you want inventory to work in ansible-runner.   There are a couple of ways this could be fixed that are mutually exclusive.

**Comment by benthomasson** [comment]
There is a fix for this in #127
